tools/multipathsecret/multipathsecret/vaultconfig.py

```diff
@@ -0,0 +1,125 @@
+import hvac
+import json
+import os
+from dataclasses import dataclass, field
+from typing import List, Dict, Optional
+
+
+@dataclass
+class Keyset:
+    '''Contains "accessor" and "id" tokens for a Vault path.
+    '''
+    accessor: str
+    id: str
+
+
+@dataclass
+class Enclave:
+    '''Maps a Vault path to 'read' and 'write' Keysets.  The 'Enclave' is
+    simply a particular vault path with its own Keysets, e.g.
+    'k8s_operator/nublado.lsst.codes'
+    '''
+    name: str
+    read: Keyset
+    write: Keyset
+
+
+@dataclass
+class VaultConfig:
+    '''Contains the vault address (a URL represented as a string), the
+    secret to be added, which is a dict mapping strings to strings
+    (but can be None, if you're deleting a secret from a path), the
+    rendered-to-memory JSON document representing all the vault paths
+    and tokens, and the list of vault paths to not update.
+
+    '''
+    vault_address: str = os.getenv('VAULT_ADDR')
+    secret: Optional[Dict[str, str]] = None
+    enclaves: List[Enclave] = field(default_factory=list)
```

Comment:
Nit: Given how it's used, I think this is crying out to be a `Dict[str, Enclave]` mapping vault paths to their enclaves.